hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/local/CleanupService.java

```diff
@@ -29,13 +29,22 @@
  */
 public final class CleanupService {
 
-    private static final long FIXED_DELAY = 60;
-    private static final long FIXED_DELAY1 = 60;
+    /**
+     * Default fixed delay in seconds for scheduled job.
+     * Visible for testing.
+     */
+    public static final long DEFAULT_FIXED_DELAY = 60;
 
+    private final long fixedDelay;
     private final String name;
     private final ScheduledExecutorService executor;
 
     public CleanupService(final String name) {
+        this(name, DEFAULT_FIXED_DELAY);
+    }
+
+    public CleanupService(final String name, final long fixedDelay) {
```

Comment:
Is this constructor only use in tests? If yes, then could you add a comment that it's test-only?

Reply:
Done.